### PR TITLE
create design_proposal.md

### PR DESCRIPTION
**Pull request by kellyyangsong**

### CODE REVIEW

**Review by jennyf19 — APPROVED** [review]
kewl! 🐩

**Review by westin-m — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/design_proposal.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+<!--
+This template is useful to build consensus about whether work should be done, and if so, the high-level shape of how it should be approached. Use this before fixating on a particular implementation.
+-->
+
+## Summary
+
+1-2 sentences. Say what this is about.
+
+## Motivation and goals
```
Comment (JoshLozensky) [inline .github/ISSUE_TEMPLATE/design_proposal.md:15]: nit: can we change goals to "success criteria" I've found this phrasing generally results in more specific and measurable goals/criteria

File: .github/ISSUE_TEMPLATE/design_proposal.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+1-2 paragraphs, or a bullet-pointed list. What existing pain points does this solve? What evidence shows it's valuable to solve this?
+
+## In scope
+
+A list of major scenarios, perhaps in priority order.
```
Comment (JoshLozensky) [inline .github/ISSUE_TEMPLATE/design_proposal.md:21]: What are "major scenarios"? Can we define this and/or provide an example?

File: .github/ISSUE_TEMPLATE/design_proposal.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+## Out of scope
+
+Scenarios you explicitly want to exclude.
```
Comment (JoshLozensky) [inline .github/ISSUE_TEMPLATE/design_proposal.md:25]: Same comment as above.... is feature an appropriate descriptor here?

**Review by JoshLozensky — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/design_proposal.md (hunk 3 above)
Comment (kellyyangsong) [reply .github/ISSUE_TEMPLATE/design_proposal.md:25]: Great comments Josh!! Will fix in later iteration!

**Pull request merged**